code.json

```diff
@@ -3,7 +3,7 @@
     "name": "ISIS3",
     "organization": "U.S. Geological Survey",
     "description": "GitHub code repository for the Integrated Software for Imagers and Spectrometers (ISIS)",
-    "version": "7.0.0",
+    "version": "7.2.0",
```

Comment:
Maybe I am missing something, but I am guessing the 7.1 tick was never ported over to dev?

Reply:
Nope, was done in the 7.1 tag